install_config/install/advanced_install.adoc

```diff
@@ -315,6 +315,34 @@ node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 ----
 ====
 
+Nodes have labels that often indicate their region. A convention is to set the
+`*region=infra*` label for the router and registry selector. For example:
+
+====
+----
+# default selectors for router and registry services
+# openshift_router_selector='region=infra'
+# openshift_registry_selector='region=infra'
+----
+====
+
+The default label is `*region=infra*`. In all other aspects, this is a normal
+node. Not all nodes are running mission-critical containers. While a node
```

Comment:
Looking again I would say this paragraph could simply be removed.  The following one covers everything I just mentioned.